source/documentation/using_the_api/api_reference.md

```diff
@@ -263,7 +263,7 @@ Parameters:
 * `limit` (Optional): Collection page size. Defaults to 100. Maximum is 5000. 
 
 ```http
-GET /entries/ HTTP/1.1
+GET /entries?start=1&limit=10 HTTP/1.1
```

Comment:
I suspect this should be `/entries/?start=1&limit=10` to keep the style consistent?